package.json

```diff
@@ -88,8 +88,8 @@
     "build:coverage": "webpack --config webpack.coverage.js",
     "build:watch": "webpack --config webpack.dev.js --watch",
     "info": "npx envinfo --system --browsers --npmPackages --binaries --languages --markdown",
-    "test": "cross-env NODE_OPTIONS=\"--max_old_space_size=4096\" karma start --single-run",
-    "test:firefox": "cross-env NODE_OPTIONS=\"--max_old_space_size=4096\" karma start --single-run --browsers=FirefoxHeadless",
+    "test": "cross-env NODE_ENV=TEST NODE_OPTIONS=\"--max_old_space_size=4096\" karma start --single-run",
```

Comment:
let's do lower case `test`. I think there's value in going by a standard that doesn't exist that will eventually pay off